fdbclient/NativeAPI.actor.cpp

```diff
@@ -224,10 +225,16 @@ void DatabaseContext::getLatestCommitVersions(const Reference<LocationInfo>& loc
 	}
 
 	if (ssVersionVectorCache.getMaxVersion() != invalidVersion && readVersion > ssVersionVectorCache.getMaxVersion()) {
-		TraceEvent(SevError, "GetLatestCommitVersions")
+		// This can happen after a recovery, GRV proxy change is detected and
+		// version cache is cleared. However, an old GRV response is processed
+		// to update the version cache with a relatively old version. Then
+		// another transaction with a newer read version from the old GRV proxy
+		// issues a read, which can trigger the condition here. Throw here to
+		// restart the transaction that will get a new GRV from the new proxy.
+		TraceEvent(SevWarn, "GetLatestCommitVersions")
 		    .detail("ReadVersion", readVersion)
-		    .detail("Version vector", ssVersionVectorCache.toString());
```

Comment:
We need to scrub our code for minor issues such as this (space in log) before merging with main, I will open a radar